### ISSUE TITLE
Being attacked by bear while standing on raw mutton.

### ISSUE THREAD

**Issue by DanielQuick** [jasonrohrer/OneLifeData7#108]
While standing on a piece of raw mutton, the hungry bear walked towards me. It ate the mutton and attacked me at the same time. I heard the sound of the bear attacking, and the bottom of my screen showed the red effect as if I was dying. However, my character model was unaffected (did not show bite marks) and I was still able to function normally (pick up items, feed, etc) leading me to assume the UI was bugged. After the typical death timer I was "Killed by Fed Grizzly Bear", with my character still appearing completely fine.

TL:DR:
Killed by fed bear at same time it was feeding
Game functioned as if I had not been bitten (except for UI/audio)
Died after the bite timer
